src/Ens.ts

```diff
@@ -115,11 +109,10 @@ export default class Ens extends EthereumNamingService {
     if (!this.isSupportedDomain(domain) || !this.isSupportedNetwork()) {
       return null;
     }
-    const nodeHash = this.namehash(domain);
     let [owner, ttl, resolver] = await this.getResolutionInfo(domain);
     if (isNullAddress(owner)) owner = null;
-    const address = await this.fetchAddress(resolver, nodeHash, EthCoinIndex);
-    const resolution = {
+    const address = await this.ignoreResolutionErrors([ResolutionErrorCode.RecordNotFound], this.fetchAddress(resolver, domain, EthCoinIndex));
+    const resolution = { 
```

Comment:
@JohnnyJumper I'd recommend installing a plugin for stripping out leading spaces to avoid unnecessary edits :)